catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/ResultIterable.java

```diff
@@ -159,6 +165,8 @@ private ResultIterable(
 
     private boolean finished = false;
 
+    private final Set<String> foundIds = new HashSet<>(2048);
```

Comment:
Two reviewers have rightly pointed out that this large set represents what we expect to be the less common case. My feeling is that the short-lived large collection is a good tradeoff for using too small a collection by default and doing too many memcopies for it to expand. @lessarderic, do you have any thoughts on this? Maybe a smaller, but still copious initial size around 250 would be a good compromise?

Reply:
Reduced initial capacity to 256.

Reply:
Changed back to 2048